integration-tests/playwright/playwright.spec.js

```diff
@@ -56,6 +56,7 @@ versions.forEach((version) => {
 
           receiver.gatherPayloadsMaxTimeout(({ url }) => url === reportUrl, payloads => {
             const events = payloads.flatMap(({ payload }) => payload.events)
+              .filter(event => event.content.type !== 'system')
```

Comment:
a bit of a nitpick but why make it two lines in this file and 3 lines in cucumber.spec.js ?

Reply:
all has been armonized.

Reply:
You mean you just deleted the lines ? hahaha i'm resolving the comment